src/main/webapp/app/apollon-diagrams/apollon-diagram-student.component.ts

```diff
@@ -55,15 +55,15 @@ export class ApollonDiagramStudentComponent implements OnInit, OnDestroy {
         }
     }
 
-    initializeApollonEditor(initialState: State) {
+    initializeApollonEditor(model: UMLModel) {
```

Comment:
Rename `initializeApollonEditor(model: UMLModel) {` to `initializeApollonEditor(initialModel: UMLModel) {`